platform/kube/inject/inject.go

```diff
@@ -190,6 +193,20 @@ func GetInitializerConfig(kube kubernetes.Interface, namespace, injectConfigName
 		return nil, err
 	}
 
+	if c.IncludeNamespaces != nil && c.ExcludeNamespaces != nil {
+		return nil, fmt.Errorf("cannot configure both namespaces and excludeNamespaces")
+	}
+
+	if c.IncludeNamespaces == nil {
+		c.IncludeNamespaces = []string{v1.NamespaceAll}
+	}
+
+	for _, excludeNamespace := range c.ExcludeNamespaces {
+		if excludeNamespace == "" {
+			return nil, fmt.Errorf("cannot configure ExcludeNamespaces as NamespaceAll")
```

Comment:
Why can't?  for example, I don't want any namespace to be injected.

Reply:
@ayj @rshriram any comments for this? According to @rshriram 's comments above, setting `excludeNamespace` as "" is same as not start initializer, so we do not need to support this value but just let user do not bother to start the initializer is good enough. 

I think @rshriram 's comments is good enough by not supporting setting `excludeNamespace` as "" , comments? @linsun

Reply:
You can use `namespace=v1.NamespaceAll` with `policy=off` or `policy=disabled` to achieve the same effect.

Reply:
ok sounds good.  Would be good to have this documented.  You may want to change the error message to indicate the action user needs to do, e.g. namespace=v1.NamespaceAll with policy=off or uninstall initializer completely.